src/GitHub.Api/Git/RepositoryManager.cs

```diff
@@ -324,6 +325,14 @@ public ITask UnlockFile(string file, bool force)
             return task.Then(ListLocks(false));
         }
 
+		public ITask CheckoutFiles( List<string> files )
+		{
+			var discard = GitClient.Discard(files);
+			discard.OnStart += t => IsBusy = true;
+			
+			return discard.Finally(() => IsBusy = false);
+		}
```

Comment:
This should be indented with spaces. There should be no spaces after `(` and before `)`